docs/UserGuide.adoc

```diff
@@ -714,40 +763,46 @@ This shows you the students in the first group of the module `CS2103`
 [NOTE]
 ====
 [horizontal]
-*Keywords are case-insensitive. eg. `cs2103t` is the same as `CS2103T`*
+*Keywords are case-insensitive. e.g. `cs2103t` is the same as `CS2103T`*
 ====
 
 
 === Session View
 
-This section explains the different commands that can be used on the session view.
+This section explains the different commands that can be used in the session view.
 
 [[AddSession]]
-==== Adding a Session `session add`
+==== Adding a Session : `session add`
 
 You can use this command to add a new session.
 
-The new session will be shown under Session View, and will automatically
+The new session will be shown in the Session View, and will automatically
 be marked as un-done.
 
+The tab that you are viewing will automatically be switched to the session
+tab as well, if you were currently on a different tab.
+
 Format: `session add m/MOD_CODE [s/START] [e/END] [d/DATE] [w/WEEKS] [t/SESSION_TYPE] [n/NOTES]`
 
 
 [TIP]
 ====
-If you want to create a session at this point of time, you can leave the start time
-`s/START`, end time `e/END` and date `d/DATE` out.
+. If you want to create a session at the current point in time, you can omit the start time
+`s/START`, end time `e/END` and date `d/DATE`.
```

Comment:
dot points here!

Reply:
instead of the other statements, say create a session with the current date and time